src/w_file.c

```diff
@@ -85,8 +85,6 @@ static void W_FILE_AddDir(w_handle_t handle, const char *path,
             I_Error("Error: opening %s", filename);
         }
 
-        I_Printf(VB_INFO, " adding %s", filename);
```

Comment:
But now it doesn't show anymore which files it added from the autoload directories. Instead, it now even lists empty directories which are in the search list. :frowning:

Reply:
What if someone will use directory with a lot of resources (some PWAD in development), than it will be huge list of files. Maybe we can increase the verbosity level for this (`VB_DEBUG`).

Reply:
Alternatively, we can print filenames from autoload directories only (it's in d_main.c)

Reply:
> Alternatively, we can print filenames from autoload directories only (it's in d_main.c)

Yes, that might be better.

Reply:
Honestly, I'd prefer if you could revert that part of the last commit. Now, it only prints which directories it looks in, but not anymore which files are loaded.

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1694/commits/8b9180426ae43f9eab63ef22283f1826a9cf2847

Reply:
Thank you!